src/content/pages/index.mdx

```diff
@@ -24,12 +25,13 @@ import TextBlock from 'components/TextBlock';
 
 <TextBlock end>
 
-## The menu bar app for controlling your microphone.
+## Mute your mic anywhere.
```

Comment:
```suggestion
## Mute your mic in any app.
```
That's what we use on the App Store:
<img width="460" alt="Screenshot 2020-05-15 at 18 58 17" src="https://user-images.githubusercontent.com/90871/82081347-0b04ee00-96de-11ea-83b2-82678505fd5e.png">

Reply:
Works for me! I kinda liked going simple with it (and to avoid conflict with "Works in every app" further below), but I'm easy. I think I was debating this one myself.